cmd/nerdctl/commit.go

```diff
@@ -39,6 +41,7 @@ func newCommitCommand() *cobra.Command {
 	}
 	commitCommand.Flags().StringP("author", "a", "", `Author (e.g., "nerdctl contributor <nerdctl-dev@example.com>")`)
 	commitCommand.Flags().StringP("message", "m", "", "Commit message")
+	commitCommand.Flags().StringSliceP("change", "c", nil, "Apply Dockerfile instruction to the created image")
```

Comment:
The help string should describe that only `CMD` instruction is supported currently.